src/Redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'categories/categoriesSlICE/CHECK_STATUS';
+
+const initialState = [];
+
+export default function categoryReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under Construction';
+
+    default:
+      return state;
+  }
+}
+
+export const checkCurStatus = () => ({
+  type: CHECK_STATUS,
```

Comment:
- [ ] The same case also applies to your categoriesSlice. Here is an example code.

```
// ./src/redux/categories/categoriesSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const categoriesSlice = createSlice({
  name: "categories",
  initialState: {
    categories: [],
  },
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
})

export const { checkStatus } = categoriesSlice.actions

export default categoriesSlice.reducer

```